### ISSUE TITLE
admin-curated blocklists

### ISSUE THREAD

**Issue by SoniEx2** [tootsuite/mastodon#13839]
### Pitch

The lack of blocklists is a feature, not a bug. However, some sort of admin-curated system would be appreciated. Anything that discourages untraceable, unaccountable external services would enable admins to vet and hold accountable discriminatory blocklists.

### Motivation

The fedi is starting to get external, untraceable and unaccountable blocking services. Additionally, instance admins may want to provide different default filtering options for different groups of ppl (e.g. kids). Having such a system, on an instance-local basis, for the admin and the instance, would be pretty good IMO. You do need to trust your admins either way so if you don't trust them to maintain a blocklist you shouldn't trust them to be your instance admin tbh.

**Comment by Tagadda** [comment]
Are you talking about this ? https://github.com/tootsuite/mastodon/pull/11298

**Comment by SoniEx2** [comment]
No, these would be additional, opt-in lists, so the admin can fit the needs of many different groups and be held accountable.

**Comment by SoniEx2** [comment]
They'd be similar to something like blocktogether, except they'd be instance-local and admin-curated. And they'd be shown to users rather than requiring a link to be shared.

So your admin could have a "Kids" blocklist. If someone is a kid, they get added to the "Kids" blocklist. When they grow up they get removed from it.

Or you could have an "Adult Content" blocklist. If someone posts adult content, they get added to the "Adult Content" blocklist. (Ideally there'd also be a removal process but I don't think any admins would actually implement such process, unfortunately. :/)

Or you could have a "Toilet Humour" blocklist. or any number of things. it just discourages ppl from grabbing random third-party lists that could silently block trans ppl or plural systems or whatnot. (the latest trend on twitter is to put plural systems on blocklists even if there's no reason for them to be on those .-.)

**Comment by Tagadda** [comment]
Maybe related issue https://github.com/tootsuite/mastodon/issues/116

**Comment by SoniEx2** [comment]
I am not sure about #116 but it seems to hint at federation?

This issue (#13839) comes from a place of not being able to meet everyone's needs. While this problem is hard, if not impossible to solve, I definitely feel like the situation could - and should - be better than it currently is.

**Comment by SoniEx2** [comment]
Additional example: Your admin could have an "Uses Ableist Slurs" blocklist. As per https://www.autistichoya.com/2014/02/violence-linguistic-ableism.html, one can pick those up due to being around ppl who use them, and struggle to avoid using them because of that. It wouldn't be unreasonable for someone to both be on such a list and actively *use* such a list, in an attempt to push those words out of their vocabulary. More importantly, if they succeed, they should be removed from the list.

This example stresses the importance of the following points:
- Being on the list and using the list aren't mutually exclusive, and there are valid reasons to use the same list you're on.
- Being on a list isn't a moral failing, nor should it be a permanent status. And I'll gladly wrestle with admins over this point. ;)

**Comment by SoniEx2** [comment]
basically a variation of this but more flexible and less authoritarian: https://mobile.twitter.com/YourFriendHau/status/1204222891614588928
